google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractStructReader.java

```diff
@@ -61,6 +61,10 @@ protected String getPgJsonbInternal(int columnIndex) {
     throw new UnsupportedOperationException("Not implemented");
   }
 
+  protected long getPgOidInternal(int columnIndex) {
```

Comment:
I think that you can remove this method (and the equivalent array methods below) as well, and instead just use the `getLongInternal` method.

Reply:
Done.